app/pdfJS.js

```diff
@@ -5,23 +5,102 @@
 
 const UrlUtil = require('../js/lib/urlutil')
 const Filtering = require('./filtering')
-const config = require('../js/constants/config')
 const appActions = require('../js/actions/appActions')
 const getSetting = require('../js/settings').getSetting
 const settings = require('../js/constants/settings')
 
-const pdfjsBaseUrl = `chrome-extension://${config.PDFJSExtensionId}/`
-const viewerBaseUrl = `${pdfjsBaseUrl}content/web/viewer.html`
+const getViewerUrl = UrlUtil.getPDFViewerUrl
```

Comment:
most of the changes below are just copied from brave/pdf.js pdfhandler.js and extension-router.js. Those scripts used to run as background scripts in the pdf.js extension but this PR deprecates them.